### DISCUSSION TITLE
etcd 3.5.15 : etcd.conf and service file, where to have variables and their format

### DISCUSSION THREAD

**Discussion by mattyb-dev** [etcd-io/etcd#18513] [Q&A]
New etcd user (for Patroni). Trying to get etcd running on RH8. We inherited V3.3 machines (RH7) and were trying to use the same config. Not having much luck.
Should the etcd.conf file use the latest and greatest yaml format for the variables? or the old ETCD_TOTO format?
Should any variables be in the etcd.service file? We have : 
WorkingDirectory, EnvironmentFile, User, ExecStart and Restart only
When declaring listen-peer-urls and listen-client-urls, do we have to put the IP and 0.0.0.0 to ensure it listens on it's own server IP address?
Apologies for the daft questions, very new to this etcd stuff.

**Comment by jmhbnz** [comment]
Hi @mattyb-dev - Thanks for your questions, please be aware etcd v3.3 is no longer supported.  You can refer to the configuration docs for that version here https://etcd.io/docs/v3.3/op-guide/configuration however any more detailed questions would be on a best efforts basis.

I note those docs include a link to a sample YAML configuration file and also mentions the format for the command line / environment flags.
